src/page/countlyBoomerangCustom.js

```diff
@@ -92,7 +92,6 @@ Plugin being used - RT, AutoXHR, Continuity, NavigationTiming, ResourceTiming
         BOOMR.subscribe('before_beacon', function (beaconData) {
           self._internals.log('[INFO]', 'Boomerang, before_beacon:', JSON.stringify(beaconData, null, 2));
           var trace = {};
```

Comment:
Nothing to do with the PR per se, but this console lock was left over and annoying